LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -272,7 +295,7 @@ function addHeader(column) {
                             //  2) Last action was a manual step move
                             //  3) No records in Step Fulfillment - Completed
                             if ( (lastActionRecord > 0)
-                                && (lastAction.stepID != 0 && lastAction.dependencyID != 0 && lastAction.actionType !== 'move')
```

Comment:
Removed reference to dependencyID here since it's not returned by the underlying database query.